tests/vmi_multus_test.go

```diff
@@ -387,6 +373,38 @@ var _ = Describe("Multus Networking", func() {
 		})
 	})
 
+	Context("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:component]VirtualMachineInstance with ovs-cni plugin interface and custom MAC address.", func() {
+		interfaces := []v1.Interface{{Name: "default", InterfaceBindingMethod: v1.InterfaceBindingMethod{Bridge: &v1.InterfaceBridge{}}},
```

Comment:
we can use only one interface here no need for the default network

Reply:
Done